### DISCUSSION TITLE
Hey!! Urgency ...

### DISCUSSION THREAD

**Discussion by Spoorthy1423** [learningequality/kolibri-design-system#751] [Q&A]
hey!! i recently raised a pr for issue number 741 in this repo, can u please merge it if the issue is solved i am a student at SST and i have a deadline today to get merged few issues. Thank you!!

**Comment by MisRob** [comment]
I'm sorry to hear that friend, but you can't expect the maintainers to do this. Generally be ready for reviews to take several days, sometimes even a week, depending on priorities. Best wishes
